File: CloudFlow/FrontEnd-App/src/app/shared/model/calendar/calendar-events-wrapper.model.ts
import { JsonObject, JsonProperty } from "json2typescript";
import { JsonCalendarEventsConverter } from "../../json-converters";
import { CalendarEventModel } from "./calendar-event.model";

@JsonObject
export class CalendarEventsWrapperModel {

	@JsonProperty("calendarEvents", JsonCalendarEventsConverter)
	public calendarEvents: CalendarEventModel[] = [];
}

File: CloudFlow/src/ro/cloudSoft/cloudDoc/presentation/client/shared/model/calendar/CalendarEventsWrapperModel.java
package ro.cloudSoft.cloudDoc.presentation.client.shared.model.calendar;

import java.util.List;

public class CalendarEventsWrapperModel {
	
	private List<CalendarEventModel> calendarEvents;

	public List<CalendarEventModel> getCalendarEvents() {
		return calendarEvents;
	}

	public void setCalendarEvents(List<CalendarEventModel> calendarEvents) {
		this.calendarEvents = calendarEvents;
	}
}